tools/gfx/d3d11/render-d3d11.cpp

```diff
@@ -2071,7 +2071,7 @@ SlangResult SLANG_MCALL createD3D11Device(const IDevice::Desc* desc, IDevice** o
     return SLANG_OK;
 }
 
-static void _initSrvDesc(IResource::Type resourceType, const ITextureResource::Desc& textureDesc, DXGI_FORMAT pixelFormat, D3D11_SHADER_RESOURCE_VIEW_DESC& descOut)
```

Comment:
Nit pick: Is there a reason to what to make impls that are private to a cpp file, not use `_`? The `_` prefixing is of course not necessary, and is more typically used for non public methods, but here it seems appropriate.

Reply:
I intended to change the name in render-d3d12.cpp but somehow the refactoring feature mistakenly changes the name here. All these functions will eventually be moved out to separate files so perhaps it is fine to keep it as is.